scripts/package-build

```diff
@@ -151,7 +151,7 @@ for pkgname in "${pkgnames[@]}"; do
             section "Stripping $pkgname"
             docker container run --interactive --rm \
                 --mount type=bind,src="$(realpath "$pkgdir")",dst=/pkg \
-                "$(image-name base)" /bin/bash \
+                "$(image-name "$image")" /bin/bash \
                 << "SCRIPT"
 find /pkg -print0 -type f | xargs --null arm-linux-gnueabihf-strip --strip-all &> /dev/null || true
```

Comment:
I suggest we do the change now instead of opening a separate PR.

```suggestion
find /pkg -print0 -type f | xargs --null "${CROSS_COMPILE}strip" --strip-all &> /dev/null || true
```

Reply:
Thanks missed that.

--- 
Btw do we also have declarations of $CC, $STRIP, $LD and such? I think they could be helpful.